README.md

```diff
@@ -25,6 +25,9 @@
         - [tomcat::config::server::service](#tomcatconfigserverservice)
         - [tomcat::config::server::tomcat_users](#tomcatconfigservertomcat_users)
         - [tomcat::config::server::valve](#tomcatconfigservervalve)
+        - [tomcat::config::context](#tomcatconfigcontext)
+        - [tomcat::config::context::resource](#tomcatconfigcontextresource)
+        - [tomcat::config::context::resourcelink](#tomcatconfigcontextresourcelink)
```

Comment:
missing `tomcat::config::server::globalnamingresources` here and elsewhere in the README